packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2337,6 +2337,19 @@ const schema: SchemaType<DbPost> = {
       }
     },
   },
+
+  ignoreRateLimits: {
+    type: Boolean,
+    optional: true,
+    nullable: true,
+    hidden: isEAForum,
+    tooltip: "Allow rate-limited users to comment freely on this post",
+    group: formGroups.moderationGroup,
+    canRead: ["guests"],
+    canCreate: ["members"],
+    canUpdate: [canUserEditPostMetadata],
```

Comment:
I think probably post field permissions should mostly just use canUserEditPostMetadata rather than doing the 'members', 'sunshineRegiment', 'admins' thing. (This allows users shared on a post with explicit edit permissions to update the relevant field, while being more specific than 'members')